lib/grp.gi

```diff
@@ -4673,6 +4674,14 @@ InstallMethod( MinimalNormalSubgroups,
     function( G )
     local nt, c, r, U;
 
+    # force an IsNilpotent check
+    # should have and IsSolvable check, as well,
+    # but methods for solvable groups are only in CRISP
+    # which aggeressively checks for solvability, anyway
+    if (not HasIsNilpotentGroup(G) and IsNilpotentGroup(G)) then
+      return MinimalNormalSubgroups( G );
```

Comment:
This line is not covered by the tests according to https://codecov.io/gh/gap-system/gap/pull/711/compare